src/content/docs/configuration/sharing.mdx

```diff
@@ -68,8 +69,8 @@ extends: organization/shared-config
 ```
 
 :::caution
-  The Mergify application must also be installed on the source repository
-  that you want to import the configuration from.
+The Mergify application must also be installed on the source repository
```

Comment:
do we want to keep the space at the beginning of the sentence in a `caution` block ?